rentsafe/rent_views/clients.py

```diff
@@ -144,7 +145,7 @@ def get_credit_color(credit_status):
                     )
                 else:
                     rate = 1
-                    if rate_ob := LeaseCurrencyRate.objects.filter(company_id= client_id).first():
+                    if rate_ob := CurrencyRate.objects.filter(user = client_id).first():
```

Comment:
consider applying the same changes
```suggestion
                    if rate_ob := CurrencyRate.objects.filter(user__company = client_id).first():
```